lib/cloudflare_client/zone/custom_hostname_v2.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'custom_hostname'
+#class CloudflareClient::Zone::CustomHostnameV2 < CloudflareClient::Zone::Base
```

Comment:
Do we need this?

Reply:
Definitely not the comment. I will change that.